browsers/opera_android.json

```diff
@@ -297,9 +297,16 @@
         "56": {
           "release_date": "2020-02-06",
           "release_notes": "https://blogs.opera.com/mobile/2020/02/easy-reading-in-opera-for-android/",
-          "status": "current",
+          "status": "retired",
           "engine": "Blink",
           "engine_version": "78"
+        },
+        "57": {
+          "release_date": "2020-03-30",
+          "release_notes": "https://blogs.opera.com/mobile/2020/03/introducing-new-features-in-opera-for-android-57/",
+          "status": "current",
+          "engine": "Blink",
+          "engine_version": "79"
```

Comment:
Should be 80?

Reply:
Oh you're right, whoops!

Reply:
oops me too (since I reviewed it…) Apologies for not checking more carefully